src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -25,10 +27,267 @@ declare global {
+     * Null if the list has not yet been populated.
+     * @defaultValue `null`
+     */
+    protected _tokenImages: string[] | null;
+
+    /**
+     * Cache the last drawn wildcard token to avoid repeat draws
+     * @defaultValue `null`
+     */
+    protected _lastWildcard: string | null;
+
     /**
      * A convenient reference to the file path of the Actor's profile image
      */
     get img(): this['data']['img'];
+
+    /**
+     * Provide an object which organizes all embedded Item instances by their type
+     */
+    get itemTypes(): Record<string, InstanceType<ConfiguredDocumentClass<typeof BaseItem>>>;
+
+    /**
+     * Test whether an Actor entity is a synthetic representation of a Token (if true) or a full Entity (if false)
+     */
+    get isToken(): boolean;
+
+    /**
+     * An array of ActiveEffect instances which are present on the Actor which have a limited duration.
+     */
+    get temporaryEffects(): InstanceType<ConfiguredDocumentClass<typeof BaseActiveEffect>>[];
+
+    /**
+     * Return a reference to the TokenDocument which owns this Actor as a synthetic override
+     */
+    get token(): InstanceType<ConfiguredDocumentClass<typeof BaseToken>> | null;
+
+    /**
+     * A convenience reference to the item type (data.type) of this Actor
+     */
+    get type(): this['data']['type'];
+
+    /** {@inheritdoc} */
+    get uuid(): string;
+
+    /**
+     * Apply any transformations to the Actor data which are caused by ActiveEffects.
+     */
+    applyActiveEffects(): void;
+
+    /**
+     * Retrieve an Array of active tokens which represent this Actor in the current canvas Scene.
+     * If the canvas is not currently active, or there are no linked actors, the returned Array will be empty.
+     * If the Actor is a synthetic token actor, only the exact Token which it represents will be returned.
+     *
+     * @param linked   - Limit results to Tokens which are linked to the Actor. Otherwise return all Tokens even those which are not linked. (default: `false`)
+     * @param document - Return the Document instance rather than the PlaceableObject (default: `false`)
+     * @returns An array of Token instances in the current Scene which reference this Actor.
+     * TODO: Configurable token class
+     */
+    getActiveTokens(linked?: boolean, document?: boolean): Token[];
+
+    /**
+     * Prepare a data object which defines the data schema used by dice roll commands against this Actor
+     */
+    getRollData(): this['data']['data'];
+
+    /** {@inheritdoc} */
+    protected _getSheetClass(): ConstructorOf<FormApplication>;
+
+    /**
+     * Create a new TokenData object which can be used to create a Token representation of the Actor.
+     * @param data - Additional data, such as x, y, rotation, etc. for the created token data (default: `{}`)
+     * @returns The created TokenData instance
+     * TODO: TokenData does not yet exist
```

Comment:
Please add a dummy for that. Then we don't have to touch `Actor` again, when it's done.